packages/Form/steps/src/StepLink.js

```diff
@@ -8,7 +8,7 @@ const propTypes = {
   ...Constants.propTypes,
   id: PropTypes.string.isRequired,
   href: PropTypes.string,
-  onClick: PropTypes.func.isRequired,
+  onClick: PropTypes.func,
```

Comment:
May be you should check if(onClick != null)

onClick(e) {
    e.preventDefault();
    const { id, href, number, title, onClick } = this.props;
    onClick({
      href,
      number,
      id,
      title,
    });
  }

Reply:
Where do you find this code code ?

Reply:
onto this file =>  packages/Form/steps/src/StepLink.js
the idea is to secure the remove of is required for all the use case.

Reply:
I'm not sure that correspond to my problem. Indeed, each time I use a Step component without an onClick props, I have an error in the console. If I add a onClick condition without removing the isRequired from the PropTypes declaration, the error will still be in the console.

Reply:
@samuel-gomez-axa , what do you think about this ?

Reply:
No problem, you can set this props optional to remove the warning.

But if you just set this props not required, the StepLink will try to call the onClick and throw an error like this : 
![image](https://user-images.githubusercontent.com/43414022/69969905-9bc78300-151d-11ea-8e14-faf97a1610d5.png)

So you have to verify if this onClick props is not null and not call if its null

Reply:
can you add an example in the story , please ?
with no onClick props : https://github.com/AxaGuilDEv/react-toolkit/tree/master/storybook/storybook/src/packages/Form/steps/src